recipes/gsw-xarray/LICENSE.rst

```diff
@@ -0,0 +1,26 @@
+Copyright 2021-2022 gsw-xarray contributors
```

Comment:
This license file is already included in the source tar, so it is not needed to be added again. We actually prefer that you use the license file from a source tar because it will be automatically updated if the license upstream changes.